src/main/java/com/pi4j/crowpi/components/SoundSensorComponent.java

```diff
@@ -0,0 +1,175 @@
+package com.pi4j.crowpi.components;
+
+import com.pi4j.context.Context;
+import com.pi4j.crowpi.components.events.DigitalEventProvider;
+import com.pi4j.crowpi.components.events.EventListener;
+import com.pi4j.crowpi.components.events.SimpleEventHandler;
+import com.pi4j.io.gpio.digital.DigitalInput;
+import com.pi4j.io.gpio.digital.DigitalInputConfig;
+import com.pi4j.io.gpio.digital.DigitalState;
+import com.pi4j.io.gpio.digital.PullResistance;
+
+/**
+ * Implementation of the CrowPi sound sensor using GPIO with Pi4J
+ */
+public class SoundSensorComponent extends Component implements DigitalEventProvider<SoundSensorComponent.SoundState> {
+    /**
+     * Pi4J digital input instance used by this component
+     */
+    protected final DigitalInput digitalInput;
+    /**
+     * Default Pin of sound sensor
+     */
+    protected static final int DEFAULT_PIN = 24;
+    /**
+     * Debounce of input in microseconds
+     */
+    protected static final long DEFAULT_DEBOUNCE = 1000;
+
+    /**
+     * Handler for simple event when a noise is recognized
+     */
+    private SimpleEventHandler onNoise;
+    /**
+     * Handler for simple event when it's silent again
+     */
+    private SimpleEventHandler onSilence;
+
+    /**
+     * Creates a new sound sensor component using the default setup.
+     *
+     * @param pi4j Pi4J context
+     */
+    public SoundSensorComponent(Context pi4j) {
+        this(pi4j, DEFAULT_PIN, DEFAULT_DEBOUNCE);
+    }
+
+    /**
+     * Create sound sensor component with custom parameters
+     *
+     * @param pi4j     Pi4J context
+     * @param address  GPIO Pin of Raspberry
```

Comment:
```suggestion
     * @param address  Custom BCM pin address
```

Or anything else, but I would avoid "of Raspberry" as it rather is the address of the sound sensor itself.